<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stepper</class>
 <widget class="QWidget" name="Stepper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="mode_groupBox">
     <property name="title">
      <string>Choose Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="m_fs_Radio">
        <property name="text">
         <string>Fullstep</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_hs_Radio">
        <property name="text">
         <string>Halfstep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_1_4_Radio">
        <property name="text">
         <string>1/4 Microstep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_1_8_Radio">
        <property name="text">
         <string>1/8 Microstep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_1_16_Radio">
        <property name="text">
         <string>1/16 Microstep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_1_32_Radio">
        <property name="text">
         <string>1/32 Microstep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_1_64_Radio">
        <property name="text">
         <string>1/64 Microstep</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_1_128_Radio">
        <property name="text">
         <string>1/128 Microstep</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="dir_groupBox">
     <property name="title">
      <string>Choose direction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="cw_radioButton">
        <property name="text">
         <string>CW</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="ccw_radioButton">
        <property name="text">
         <string>CCW</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="steps_groupBox">
     <property name="title">
      <string>Add steps</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="cont_Radio">
        <property name="text">
         <string>Continuos</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="steps_Radio">
        <property name="text">
         <string>Steps:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="steps_lineEdit">
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="set_groupBox">
     <property name="title">
      <string>Choose motor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="m1_pushButton">
        <property name="text">
         <string>MOTOR 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m2_pushButton">
        <property name="text">
         <string>MOTOR 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="control_groupBox">
     <property name="title">
      <string>Test rotation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="start_PB">
        <property name="text">
         <string>START</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="stop_PB">
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
